src/style.css

```diff
@@ -2,0 +1,78 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+input {
+  outline: none;
+  border: none;
+}
+
+.search {
+  display: flex;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+  box-sizing: border-box;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search:focus-within {
+  border-color: rgba(225, 231, 237, 1);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 800;
```

Comment:
Using `font-weight: 800;` here might not have any effect if the font file for this weight is not loaded. Ensure that the corresponding font file is available or adjust the weight to match the loaded fonts.